roles/encryption_configure/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+    # Enabling Encryption for IBM Spectrum Scale (GPFS) using GKLM - Security Guardium Lifecycle Manager
+
+    - name: Executing common tasks on the cluster
+      block:
+        # Import the 'check_filesystem.yml' task for checking the filesystem in the cluster.
+        - import_tasks: check_filesystem.yml
+        # Import the 'add_host_entry.yml' task for adding the keyservers to the host file of the cluster.
+        - import_tasks: add_host_entry.yml
+        # Import the 'add_sklm_keyserver.yml' task for adding the key server to the cluster
+        - import_tasks: add_sklm_keyserver.yml
+        # Import the 'create_tenant.yml' task for creating a tenant to the keyserver for cluster.
+        - import_tasks: create_tenant.yml
+        # Import the 'create_client.yml' task for creating a client to the keyserver for cluster.
+        - import_tasks: create_client.yml
+        # Import the 'register_client.yml' task for registerin the client to the tenant.
+        - import_tasks: register_client.yml
+      when: 
+        - (is_admin_node == True and scale_cluster_type == "storage") or 
```

Comment:
Do we still need `is_admin_node`

Reply:
Yes we require to execute in one node. This helps to avoid creating a different inventory or group for it.